fuse.py

```diff
@@ -432,6 +442,8 @@ class as is to Operations, instead of just the fh field.
         self.operations = operations
         self.raw_fi = raw_fi
         self.encoding = encoding
+        
+        self.with_nanosecond_int = hasattr(operations, 'WITH_NANOSECOND_INT')
```

Comment:
Similar comment to the `NANOSECOND_INT_AVAILABLE` one above (for similar reasons); use `getattr(fuse, 'WITH_NANOSECOND_INT', None)` instead of `hasattr(fuse, 'WITH_NANOSECOND_INT')`.